extensions/dicom/package.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "dicom",
+  "title": "dicom",
+  "description": "dicom tag lookup",
+  "icon": "dicom.png",
+  "author": "sedghi",
+  "categories": [
+    "Data",
+    "Documentation",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "dicom",
```

This should relate to the 
```suggestion
      "name": "dicom",
```